docs/guides/cluster-to-advanced-cluster-migration-guide.md

```diff
@@ -0,0 +1,181 @@
+
+## Main Changes Between `mongodbatlas_cluster` and `mongodbatlas_advanced_cluster`
+
+1. Replication Spec Configuration: Supports different node types (electable, analytics, read_only) where hardware configuration can differ between node types.
+2. Provider Settings: Moved from the top level to the replication spec allowing you to create multi-cloud clusters.
+3. Auto Scaling: Moved from the top level to the replication spec allowing you to scale replication specs individually.
+4. Backup Configuration: Renamed from `cloud_backup` to `backup_enabled`.
+5. See the [Migration Guide: Advanced Cluster New Sharding Schema](advanced-cluster-new-sharding-schema#migration-sharded) for changes to `num_shards` and the new `zone_id`.
+
+### Example 1: Old Configuration (`mongodbatlas_cluster`)
+
+```terraform
+resource "mongodbatlas_cluster" "this" {
+  project_id   = var.project_id
+  name         = "legacy-cluster"
+  cluster_type = "REPLICASET"
+
+  provider_instance_size_name = "M10" # 1 Replication Spec Configuration
+  provider_name               = "AWS" # 2 Provider Settings
+
+  auto_scaling_disk_gb_enabled = true # 3 Auto Scaling
+  cloud_backup                 = true # 4 Backup Configuration
+
+  replication_specs {
+    num_shards = 1
+    regions_config {
+      region_name     = "US_EAST_1"
+      priority        = 7
+      electable_nodes = 3 # 1 Replication Spec Configuration
+      analytics_nodes = 1 # 1 Replication Spec Configuration
+      read_only_nodes = 0 # 1 Replication Spec Configuration
+    }
+  }
+}
+```
+
+### Example 2: New Configuration (`mongodbatlas_advanced_cluster`)
+
+```terraform
+resource "mongodbatlas_advanced_cluster" "this" {
+  project_id     = var.project_id
+  name           = "advanced-cluster"
+  cluster_type   = "REPLICASET"
+  backup_enabled = true # 4 Backup Configuration
+
+  replication_specs {
+    region_configs {
+      auto_scaling { # 3 Auto Scaling
+        disk_gb_enabled = true
+      }
+      region_name   = "US_EAST_1"
+      priority      = 7
+      provider_name = "AWS" # 2 Provider Settings
+
+      electable_specs { # 1 Replication Spec Configuration
+        instance_size = "M10"
+        node_count    = 3
+      }
+      analytics_specs { # 1 Replication Spec Configuration
+        instance_size = "M10"
+        node_count    = 1
+      }
+    }
+  }
+}
+```
+
+### Output Changes
+
+- `container_id`:
+  - Before: `mongodbatlas_cluster.this.replication_specs[0].container_id` was a flat string, such as: `669644ae01bf814e3d25b963`
+  - After: `mongodbatlas_advanced_cluster.this.replication_specs[0].container_id` is a map, such as: `{"AWS:US_EAST_1": "669644ae01bf814e3d25b963"}`
+  - If you have a single region you can access the `container_id` directly with: `one(values(mongodbatlas_advanced_cluster.this.replication_specs[0].container_id))`
+
+## Best Practices Before Migrating
```

Comment:
Should the period be inside the link? I am not familiar with this syntax. Just checking because I've seen it a few times in this document. In rendered text, the period should be after the link.